content/00.build/30.zksync-sso/30.transaction-flow.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Transaction Flow
+description: Understand transaction flows when using ZKsync SSO.
+---
+
+This section outlines the flow for user authentication and session management within an application using ZKsync SSO.
+
+### 1. User Initiates Sign-In
+
+- **Action**: The user navigates to an app featuring the **"Sign in with ZKsync"** option.
+
+### 2. Authentication Options
+**Sign Up:**
+
+- **Action**: User is prompted to create a passkey (e.g., Face ID or Touch ID)
+
+- **Process**: A new smart contract account is deployed by ZKsync SSO Auth Server using a shared, subsidized factory.
+
+**Sign In:**
+
+- **Action**: User is prompted to enter a passkey (e.g., Face ID or Touch ID)
+
+### 3. Application requests a session (optional)
+
+- **Action**: ZKsync SSO presents the permissions and actions to be granted, including:
+  - Spend limits
+  - Session key duration
+  - Custom module installations
```

Comment:
I don't think this is a good example of what will be presented during session creation.

More like:
- allowed methods to call
- allowed arguments to pass
- allowed targets to transfer funds to
- limits restricting spent funds & passed arguments